packages/deprecation-crawler/sandbox/deprecations/raw-deprecations.json

```diff
@@ -1,9 +1,9 @@
 {
     "version": "master",
-    "date": "sandbox-date",
```

Comment:
This change is here because the git command in sandbox mode return an empty string